lexnlp/ml/catalog/tests/test_catalog_path.py

```
@@ -104,17 +102,56 @@ def call_cached():
+    import threading
+    import lexnlp.ml.catalog as catalog
+
+    monkeypatch.setattr(catalog, "CATALOG", tmp_path)
+    catalog.invalidate_catalog_cache()
+
+    errors: list[Exception] = []
+
+    def do_invalidate():
+        try:
+            catalog.invalidate_catalog_cache()
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)